### ISSUE TITLE
Armour questions

### ISSUE THREAD

**Issue by damienhurrell** [kdechant/eamon#54]
Now that ER allows more armour types, I thought I'd have a look around for some options.

The first item I thought of was the crown in the beginner's forest:  I've travelled there, freed the fairy but when I try to wear it I get the error message "Error: Cannot read property 'id' of null"

If I then pick it up I can wear it, but it doesn't change my armour class, and I don't retain it when I get back to the main hall.  I don't know if you intended for it to be armour, but right now it isn't. 

So next I went to the Cave of the Mind: there's a helmet there that I could wear, but it also doesn't seem to be armour. It has special qualities in the adventure but it's still a hat made of metal: it would probably protect a player at least a little.

Do the attributes of these items need to be updated?  Or do you think they are not armour?

**Comment by damienhurrell** [comment]
I note the helmet in the assault on the Clone master has an AC of 1 and is retained when returning to the hall.

**Comment by kdechant** [comment]
The "wear crown" error in the Beginner's Forest is fixed. (You do still have to pick it up first.)

I also gave an armor type and armor class to the other items you mentioned. Those adventures were ported before I added the capability of having armor types other than body armor and shields. I never went back and updated the items, but it makes sense now that I think about it. The combat in some adventures is certainly tough enough that the extra 1 AC will be useful.

BTW, I added the helmet armor type while I was writing Clone Master, so that's why that one was set up with an AC of 1.
